actors/miner/src/partition_state.rs

```diff
@@ -822,6 +822,37 @@ impl Partition {
         self.validate_bf_state()?;
         Ok(())
     }
+
+    // ajust epoch info of expirations_epochs and early_terminated within `Partition`
+    pub fn adjust_for_move<BS: Blockstore>(&mut self, store: &BS) -> anyhow::Result<()> {
```

Comment:
Can't we delete this entire function now?

Reply:
Note: now that we're not re-quantizing, we need to fix 

https://github.com/filecoin-project/builtin-actors/blob/6b2c94fde8a0db9815f143081cd42bd96ab9f760/actors/miner/src/expiration_queue.rs#L774

Instead of looking up the quantized expiration, we need to scan the range `quantize_down(sector.expiration)+1` through `quantize_up(sector.expiration)`.

Reply:
@Stebalien deleted the no-longer-useful function , good catch!

About `find_sectors_by_expiration` it seems there's logic that already handles this [here](https://github.com/filecoin-project/builtin-actors/blob/6b2c94fde8a0db9815f143081cd42bd96ab9f760/actors/miner/src/expiration_queue.rs#L811)?

Reply:
Hm. You're right, although:

1. I'm not sure why we have that code... Given the way this used to work, it should never have triggered. I.e., sectors should either (a) expire at their quantized expiration or (b) be scheduled for early termination. But that code explicitly _doesn't_ handle sectors scheduled for early termination.
2. That code will be _slow_ as it'll have to scan everything instead of _just_ scanning the correct ranges.

Reply:
I.e., the comment:

https://github.com/filecoin-project/builtin-actors/blob/f7b7bd7af6d1981c56db415175492163a5c099dc/actors/miner/src/expiration_queue.rs#L807-L808

Conflicts with:

https://github.com/filecoin-project/builtin-actors/blob/f7b7bd7af6d1981c56db415175492163a5c099dc/actors/miner/src/expiration_queue.rs#L819-L822

Reply:
About the performance, I think it should not be a problem as long as `for_each_while` iterates from lower to higher? This method is also used in [here](https://github.com/filecoin-project/builtin-actors/blob/6b2c94fde8a0db9815f143081cd42bd96ab9f760/actors/miner/src/bitfield_queue.rs#L117), which is also assuming the iteration order.

Reply:
Now for each active sector, `[sector.expiration, sector.expiration+EPOCHS_IN_DAY]` will be scanned.